miner/worker.go

```diff
@@ -81,7 +81,11 @@ const (
 )
 
 var (
-	commitTxsTimer = metrics.NewRegisteredTimer("worker/committxs", nil)
+	commitTxsTimer        = metrics.NewRegisteredTimer("worker/committxs", nil)
```

Comment:
Is it possible to move these to a separated file, as well as several other files? It will be easier to use when others want to apply our code.